user/enterprise/setting-up-travis-ci-enterprise.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Setting up Travis CI Enterprise
+layout: en_enterprise
+redirect_from:
+  - /user/enterprise/installation/
+  - /user/enterprise/prerequisites/
+  - /user/enterprise/install-on-xenial/
+
+---
+
+Travis CI Enterprise works with your GitHub.com or GitHub Enterprise setup.
+
+## Prerequisites
+
+  * A valid Travis CI Enterprise license or [trial license](https://enterprise.travis-ci.com/signup). If you're interested in using [High Availability](/user/enterprise/high-availability/), [please let us know](mailto:enterprise@travis-ci.com) so we can get your trial license configured.
+
+  * At least two virtual machines on your private cloud that meet the [system requirements](#system-requirements)
+  * The `secret` and `clientid` of a GitHub.com or GitHub Enterprise [OAuth application](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/registering-oauth-apps/) configured with:
+    - *Homepage URL* - `https://travis-ci.<your-domain>.com`
```

Comment:
To be honest these URL examples were a little confusing to me, both as a customer (at my last job) and as a new developer. I thought that it was somehow required to use `travis-ci.` as a prefix and only after ignoring it did things work.

Could we perhaps do something like:

```
- *Homepage URL* - `https://<your-domain>`
- *Authorization callback URL* - `https://<your-domain>/api`
```

Also: the current docs also specifically say that you must not include a trailing slash. Is that no longer a requirement? @schultyy Do you know?

Reply:
Also: I think a newline before the URL example list is needed:

<img width="939" alt="screen shot 2018-12-12 at 10 04 18 am" src="https://user-images.githubusercontent.com/45571088/49882044-5d5d2380-fdf5-11e8-803d-25bd275aa7d5.png">

In the current docs we include a newline, fwiw.

Reply:
@philpeble Can confirm, no trailing slash for the domains, otherwise authentication doesn't work.

Reply:
Great, thanks. Then @plaindocs I think we should add that line back.